src/App.js

```diff
@@ -3,17 +3,13 @@ import { Switch } from 'react-router-dom';
 import Header from './components/Header';
 import { privateRouteComponents, routeComponents } from './constants/routes';
 
-
 export default class App extends Component {
   render() {
     return (
-      <>
-        
-        <div className="App">
+      <div className="App">
         <Header />
-        <Switch> {[...privateRouteComponents, ...routeComponents]} </Switch>
-        </div>
-      </>
+        <Switch>{[...privateRouteComponents, ...routeComponents]}</Switch>
```

Comment:
not the best approach. It will fail once i ask you to make part of the private routes to be lazy imported 
Much better to declare them explicitly